github/ipld.yml

```diff
@@ -110,9 +110,6 @@ repositories:
     advanced_security: false
     allow_update_branch: false
     archived: false
-    collaborators:
-      admin:
-        - vmx
```

Comment:
Why am I removed? I created that spec and repo. What does it mean if there are no admins? I guess me having push access would still make sense?